src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1396,6 +1402,25 @@ internal static bool OnServerCertificateCustomValidationCallback(object sender,
             return false;
         }
 
+        private static void ThrowNameMismatchException(object sender, X509Certificate cert)
+        {
+            var hostname = ((HttpRequestMessage)sender).RequestUri.DnsSafeHost;
+            var cert2 = new X509Certificate2(cert.Handle);
```

Comment:
@ayende This is a difference between 4.0 and 4.1
Netstandard1.3 doesn't have a ctor which accepts x509certificate so using the handle instead.